tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
+    "resolveJsonModule": true,
```

Comment:
This seems unrelated.

Reply:
> snook in a tsconfig change which I need for

as per RWI

Reply:
Please put it in its own PR so there's a bit more context for it, such as what it actually fixes - it's not obviously clear here.